cmd/openshift-tests/images.go

```diff
@@ -229,9 +230,22 @@ func pulledInvalidImages(fromRepository string) ginkgo.JUnitForEventsFunc {
 
 		allowedPrefixes := allowedPrefixes.List()
 
+		// these are the possible format we need to work with:
+		// 1. reason/Pulled Container image "quay.io/openshift/community-e2e-images:e2e-7-k8s-gcr-io-e2e-test-images-busybox-1-29-2-cqcP1Tnbm-JjJyUA" already present on machine
+		// 2. reason/Pulled image/quay.io/openshift/community-e2e-images:e2e-7-k8s-gcr-io-e2e-test-images-busybox-1-29-2-cqcP1Tnbm-JjJyUA
+		// the regexp tries to match either image/(group) or image "(group)",
+		// where (group) is constructed from three subgroups divided with /
+		// where each has one or more characters from these:
+		// \w (word characters - [0-9A-Za-z_]), -, :, . (needs escaping)
+		imageRe, err := regexp.Compile(`image/([\w-:/\.]+/[\w-:/\.]+/[\w-:/\.]+)$|image "([\w-:/\.]+/[\w-:/\.]+/[\w-:/\.]+)"`)
```

Comment:
why is `/` in the character class?

i'd expect `[\w-:\.]+`

Reply:
Good catch, it should've been removed when switching to three subgroups splitted with `/` - updating.